app/views/posts/show.html.erb

```diff
@@ -8,6 +8,11 @@
   </div>
   <div>
     <p><%=@post.text%></p>
+  </div>
+  <div class="post">
+    <% if can? :delete, @post %>
+      <%= link_to 'Delete', user_post_path(@current_user, @post), method: :delete, data: { confirm: 'Are you sure?' } %>
+    <% end %>
   </div>
     </div>
     <% if  !@post.comments.empty?%>
```

Comment:
Also, the comments counter is not updated successfully after deleting comments. In the Screenshot below I added 4 comments and then deleted 2. But the comments counter still displays 4.

![image](https://github.com/AyoMoses1/Gems_Blog/assets/38088818/941d204f-52d1-4971-a311-b84c1a0ef8d9)

Reply:
This change has been done well